Gemfile

```diff
@@ -64,9 +64,7 @@ group :test do
 
   # For better test reporting in CircleCI
   # http://blog.circleci.com/announcing-detailed-test-failure-reporting/
-  # with a fix from
-  # https://github.com/circleci/rspec_junit_formatter/pull/4
-  gem 'rspec_junit_formatter', git: 'https://github.com/amitree/rspec_junit_formatter.git', ref: '33a0fdd'
+  gem 'rspec_junit_formatter'
```

Comment:
[The CircleCI fork](https://github.com/circleci/rspec_junit_formatter) now recommends using the canonical gem. All good!